server/query/user_screen.js

```diff
@@ -0,0 +1,19 @@
+const db = require("../connection/pg");
+
+module.exports = {
+  async create({ screen_canvas_id, user_billboard_id, user_screen_name, description, config }) {
+    let query = "INSERT INTO user_screens (screen_canvas_id, user_billboard_id, user_screen_name, description, config) ";
+    query += `VALUES ('${screen_canvas_id}', '${user_billboard_id}', '${user_screen_name}', '${description}', '${config}')`;
+    return db.query(query);
+  },
+
+  async fetchById(id) {
+    const userScreens = await db.query(`SELECT * FROM user_screens WHERE id = '${id}'`);
```

Comment:
not sure if this is a concern but should we be using the parameterized queries to prevent possible SQL injection?

I found this link that talks about postgres & node.js that may be helpful

https://stackoverflow.com/questions/58174695/prevent-sql-injection-with-nodejs-and-postgres

Reply:
That's a good idea! I initially didn't think our security was a concern. When I asked flourin this early on he said it didn't matter much. Let's have this done on a separate ticket like a `task` or a `subtask` to change them all to do this.